### PR TITLE
Added and categorized input devices

### PR DESCRIPTION
**Pull request by Y2nn1c**
Made a division based on what is actually used or not.
Would like to delete the "for fun" section entirely since those input devices are no used by anyone aside for a funny video from a content creator or two.
However I don't know where I would put them instead so I will leave them here for now.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] Add the keypad and make a new article for it
- [x] Finish the keypad article
SKIP_OUTDATED_CHECK

### CODE REVIEW

File: wiki/Gameplay/Input_device/en.md [hunk 1]
```diff
@@ -8,11 +8,14 @@ An **input device** is a device that is used to control input on a computer or s
...
-- [Graphics tablet](/wiki/Gameplay/Input_device/Graphics_tablet)
-- [Wiimote](/wiki/Gameplay/Input_device/Wiimote)
+
+Uncommon input devices mostly used for fun:
+
+- [Beatmania controller](/wiki/Gameplay/Input_device/Beatmania_controller)
 - [Drum controller](/wiki/Gameplay/Input_device/Drum_controller)
 - [Dance pad](/wiki/Gameplay/Input_device/Dance_pad)
-- [Beatmania controller](/wiki/Gameplay/Input_device/Beatmania_controller)
+- [Wiimote](/wiki/Gameplay/Input_device/Wiimote)
```
[the remark this answers is not in the document]
Comment (Y2nn1c) [reply wiki/Gameplay/Input_device/en.md]: Got it, ty

File: wiki/Gameplay/Input_device/en.md [hunk 2]
```diff
@@ -8,11 +8,15 @@ An **input device** is a device that is used to control input on a computer or s
...
-- [Drum controller](/wiki/Gameplay/Input_device/Drum_controller)
-- [Dance pad](/wiki/Gameplay/Input_device/Dance_pad)
+
+Uncommon input devices mostly used for fun:
+
 - [Beatmania controller](/wiki/Gameplay/Input_device/Beatmania_controller)
+- [Dance pad](/wiki/Gameplay/Input_device/Dance_pad)
+- [Drum controller](/wiki/Gameplay/Input_device/Drum_controller)
+- [Wiimote](/wiki/Gameplay/Input_device/Wiimote)
+
```
Comment (MChecaH) [inline wiki/Gameplay/Input_device/en.md:22]: Removed extra newline.
```suggestion
```

**Comment by peppy** [comment]
this seems pointless and subjective

**Comment by Y2nn1c** [comment]
> this seems pointless and subjective

what part exactly, if you want i will just take the PR down

**Comment by Y2nn1c** [comment]
> this seems pointless and subjective

what do you mean by that, please elaborate

**Comment by cl8n** [comment]
IMO splitting the "common devices" list into common and uncommon is good, since it's just wrong at the moment. I don't think you need to further split it into "mostly used for fun" though, that is going into "too subjective" territory like peppy said. I also think keypad still fits better within the keyboard article as it already was, because it's just a small keyboard, but the new image is fine

**Comment by peppy** [comment]
The "mostly for fun" inputs are valid and supported devices. We don't need to decide what is common and what isn't. Just list them all and let the users decide.

I'd put the most common ones at the top, but no subjective headings required. Not going to be helpful to anyone.

**Pull request closed without merging**